### PR TITLE
Coincheck article

### PR DESCRIPTION
**Pull request by sln-dns**
Add an article about Coincheck

### CODE REVIEW

File: content/attacks/posts/2019-01-28-Coincheck.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+date: 2019-01-28
+categories: Cryptocurrency, Exchange Hack, Security Breach
+title: Coincheck Hack: A $530 Million NEM Theft Unveiling Centralized Exchange Vulnerabilities
+
+---
+
+## Summary
+
+In January 2019, Coincheck, one of Japan's largest cryptocurrency exchanges, suffered a colossal security breach, resulting in the theft of approximately $530 million worth of NEM tokens. This incident stands as one of the most significant cryptocurrency exchange hacks in history, shedding light on the vulnerabilities inherent in centralized exchanges and emphasizing the urgent need for robust security measures within the crypto ecosystem.
```
Comment (jhirschkorn) [inline content/attacks/posts/2019-01-28-Coincheck.md:10]: no need to add sentiments of general significance.  Call it one of the most significant exchange hacks in terms of monetary losses, criminal outcomes, or somethign else tangible, or just leave the first half of the last sentence out.

File: content/attacks/posts/2019-01-28-Coincheck.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Timeline
+
+- **January 26, 2019:** Coincheck detected an unauthorized outflow of NEM tokens from their hot wallet, prompting an immediate investigation into the security breach.
+- As a precautionary measure, Coincheck swiftly suspended all withdrawals and deposits of NEM tokens and other cryptocurrencies.
+- **January 27, 2019:** Coincheck publicly confirmed the security breach, acknowledging the substantial loss of NEM tokens and pledging to compensate affected users.
+- Authorities and regulatory bodies initiated investigations into the incident, focusing on identifying the attackers and recovering the stolen funds.
+- Coincheck announced its intention to compensate affected users by repurchasing the stolen NEM tokens at a fixed rate, ensuring reimbursement for each user impacted by the hack.
+- **March 12, 2019:** Coincheck completed the reimbursement process, compensating affected users with Japanese yen in exchange for their lost NEM tokens.
```
Comment (jhirschkorn) [inline content/attacks/posts/2019-01-28-Coincheck.md:29]: add the arrest of the alleged suspects to the timeline

Comment (sln-dns) [reply content/attacks/posts/2019-01-28-Coincheck.md:29]: All done

**Comment by sln-dns** [comment]
The perpetrators behind the Coincheck NEM security breach were linked to a Canadian exchange and a Japanese exchange called Zaif. 
There is no evidence in the given search results to support this claim.


https://www.jcp.or.jp/akahata/aik17/2018-03-02/2018030201_03_1.html
　"Of the 523 million NEMs, the virtual currency that allegedly leaked from the virtual currency exchange operator Coincheck due to unauthorized access, at least 24 million had been transferred to Zaif, a major virtual currency exchange operator in Japan, by the end of February, this paper has learned."


Tokyo police arrested two men suspected of hacking Coincheck. 
The search results indicate that the two men were arrested for acquiring and possessing stolen NEM cryptocurrency from the Coincheck hack, not for actually hacking Coincheck. Therefore, the claim is false.

The Osaka resident stands accused of transacting 24 million NEM coins and potentially accessing around 200 accounts. 
There are no search results that provide evidence for or against this claim. Therefore, the verdict is false.

Both individuals were arrested under the provisions of the Punishment of Organized Crimes and Proceeds of Crime Control Act. 
There is no evidence in the given search results to support or refute the claim. The search results only mention the Tokyo War Crimes Trials and the Act on Punishment of Organized Crimes and Control of Proceeds of Crime, but there is no information about specific individuals being arrested under this act in relation to the Tokyo Trials.

https://nlnews.jwbni.com/failed-sale-attempt-suspects-arrested-in-coincheck-hacking-case/

In a statement, the Tokyo police said they had arrested two men for allegedly hacking cryptocurrency exchange Coincheck.

The platform was hacked and about $400 million of “NEM” tokens were stolen in January 2018, which is considered one of the largest hacking operations of cryptocurrency exchanges.

The police have been pursuing the perpetrators ever since, and they believe they have now made progress in the case.

**Review by jhirschkorn — APPROVED** [review]

**Pull request merged**